tools/generate-rust-dashboards/src/team_config.rs

```diff
@@ -0,0 +1,148 @@
+                    }
+                    .into(),
+                    DistributionMetric {
+                        kind: DistributionMetricKind::Timing,
+                        display_name: "Places run_maintenance_prune_time() time",
+                        ping: "metrics",
+                        category: "places_manager",
+                        metric: "run_maintenance_prune_time",
+                        axis_label: "time (ms)",
+                        value_divisor: Some(1_000_000),
+                        value_filter: None,
+                        applications: vec![Android],
+                        link_to: None,
+                    }
+                    .into(),
+                    DistributionMetric {
+                        kind: DistributionMetricKind::Timing,
+                        display_name: "Places run_maintenance_vacuum_time() time",
+                        ping: "metrics",
+                        category: "places_manager",
+                        metric: "run_maintenance_vacuum_time",
+                        axis_label: "time (ms)",
+                        value_divisor: Some(1_000_000),
+                        value_filter: None,
+                        applications: vec![Android],
+                        link_to: None,
+                    }
+                    .into(),
+                ],
+            }],
+        },
+        TeamConfig {
+            team_name: "DISCO",
+            components: vec![Component::Suggest],
+            component_errors: true,
+            sync_metrics: false,
+            main_dashboard_metrics: vec![
+                LabeledDistributionMetric {
+                    kind: DistributionMetricKind::Timing,
+                    display_name: "Suggest ingest time",
+                    ping: "metrics",
+                    category: "suggest",
+                    metric: "ingest_time",
+                    axis_label: "time (ms)",
+                    value_divisor: Some(1_000_000),
+                    value_filter: Some(100_000),
+                    applications: vec![Desktop],
+                }
+                .into(),
+                LabeledDistributionMetric {
+                    kind: DistributionMetricKind::Timing,
+                    display_name: "Suggest ingest download time",
+                    ping: "metrics",
+                    category: "suggest",
+                    metric: "ingest_download_time",
+                    axis_label: "time (ms)",
+                    value_divisor: Some(1_000_000),
+                    value_filter: None,
+                    applications: vec![Desktop],
+                }
+                .into(),
+                LabeledDistributionMetric {
+                    kind: DistributionMetricKind::Timing,
+                    display_name: "Suggest query time",
+                    ping: "metrics",
+                    category: "suggest",
+                    metric: "ingest_download_time",
```

Comment:
Is this the right metric here? it's the same as `Suggest ingest download time` above, lets either put the query_time param here or just remove this